Cargo.toml

```diff
@@ -9,15 +9,15 @@ repository = "https://github.com/maidsafe/safe_crypto"
 version = "0.2.0"
 
 [dependencies]
+lazy_static = { version = "~1.0.1", optional = true }
```

Comment:
Let me be picky just this once, maybe you'll like it :)
Seems like you've used  cargo edit` to reformat `Cargo.toml`. Such change could be commited separately.
See "Make your commits atomic" at https://maidsafe.atlassian.net/wiki/spaces/BT/pages/170917967/Git+best+practices :)